### PR TITLE
Releng issue for move to GitHub #2

### PR DESCRIPTION
**Pull request by ruspl-afed**
Add initial Jenkins

Signed-off-by: Alexander Fedorov <alexander.fedorov@arsysop.ru>

### CODE REVIEW

File: NOTICE [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+## Source Code
+
+The project maintains the following source code repositories:
+
+* https://github.com/eclipse-mylyn/org.eclipse.mylyn.git
+* https://github.com/eclipse-mylyn/org.eclipse.mylyn.all.git
+* https://github.com/eclipse-mylyn/org.eclipse.mylyn.builds.git
+* https://github.com/eclipse-mylyn/org.eclipse.mylyn.commons.git
+* https://github.com/eclipse-mylyn/org.eclipse.mylyn.context.git
+* https://github.com/eclipse-mylyn/org.eclipse.mylyn.context.mft.git
```
Comment (BeckerFrank) [inline NOTICE:34]: I think that repository no longer active.
It is not included in the Mylyn.setup.

File: NOTICE [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+* https://github.com/eclipse-mylyn/org.eclipse.mylyn.docs.git
+* https://github.com/eclipse-mylyn/org.eclipse.mylyn.docs.intent.main.git
+* https://github.com/eclipse-mylyn/org.eclipse.mylyn.docs.vex.git
+* https://github.com/eclipse-mylyn/org.eclipse.mylyn.incubator.git
```
Comment (BeckerFrank) [inline NOTICE:38]: I think that repository no longer active.
It is not included in the Mylyn.setup.

File: NOTICE [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+* https://github.com/eclipse-mylyn/org.eclipse.mylyn.reviews.git
+* https://github.com/eclipse-mylyn/org.eclipse.mylyn.reviews.ui.git
```
Comment (BeckerFrank) [inline NOTICE:40]: I think that repository no longer active.
It is not included in the Mylyn.setup.

File: NOTICE (hunk 1 above)
Comment (ruspl-afed) [reply NOTICE]: Done

File: NOTICE (hunk 2 above)
Comment (ruspl-afed) [reply NOTICE]: Done

File: NOTICE (hunk 3 above)
Comment (ruspl-afed) [reply NOTICE]: Done

**Comment by ruspl-afed** [comment]
Now it fails because tests are running for too long. The plan is to remove tests that we already passed during component testsing.

**Comment by ruspl-afed** [comment]
After removing all the tests that we already running for others it converged. Merging

**Pull request merged**